Vwitter/View Controllers/SelectAudienceViewController.m

```diff
@@ -0,0 +1,152 @@
+//
+//  SelectAudienceViewController.m
+//  Vwitter
+//
+//  Created by Christina Li on 7/8/22.
+//
+
+#import "SelectAudienceViewController.h"
+#import <Parse/Parse.h>
+#import "Follow.h"
+#import "AudienceMemberCell.h"
+#import "Vent.h"
+#import "VentAudience.h"
+#import <MBProgressHUD/MBProgressHUD.h>
+
+@interface SelectAudienceViewController () <UITableViewDataSource, UITableViewDelegate>
+@property (weak, nonatomic) IBOutlet UIButton *postVentButton;
+@property (weak, nonatomic) IBOutlet UITableView *tableView;
+@property (strong, nonatomic) NSMutableArray *arrayOfAudienceMembers;
+@property (strong, nonatomic) NSMutableArray *arrayOfSelectedAudience;
```

Comment:
Does the order actually matter? `removeObject:` on `NSMutableArray` is potentially slow (O(n), it has to iterate through each item in the array to find it). Instead, you could make this an `NSMutableSet`.